client/cody/webviews/UserHistory.module.css

```diff
@@ -0,0 +1,69 @@
+.heading-container {
+    display: flex;
+    flex-direction: row;
+    margin: 1rem;
+    align-items: center;
+}
+
+.heading-container h3 {
+    margin: 0;
+}
+
+.clear-button-container {
+    margin-left: auto;
+}
+
+.items-container {
+    margin: 0 1rem;
+    display: flex;
+    gap: 1rem;
+    flex-direction: column;
+}
+
+/**
+    Sadly we have to use our own button instead of VSCodeButton because it has a
+    #shadow-root style with max-width: 300px AND we get a weird border when the
+    window is wide. So we use a <button> and set our own styles (mostly found by
+    inspecting the standard button)
+**/
+.item-button {
+    all: unset;
+    box-sizing: border-box;
+    display: inline-flex;
+    justify-content: center;
```

Comment:
```suggestion
    justify-content: flex-start;
    width: 100%;
```
To fix centered text issue:

<img width="565" alt="image" src="https://github.com/sourcegraph/sourcegraph/assets/68532117/a5488447-0a71-4ddc-8f3d-4cb3116df385">